EIPS/eip-3770.md

```diff
@@ -4,7 +4,7 @@ title: Chain-specific addresses
 description: A standard for displaying CAIP-10 account identifiers in a human readable format
 author: Lukas Schor (@lukasschor), Richard Meissner (@rmeissner), Pedro Gomes (@pedrouid), ligi <ligi@ligi.de>
 discussions-to: https://ethereum-magicians.org/t/chain-specific-addresses/6449
-status: Draft
+status: Final
```

Comment:
Proposals normally go through the `Review`, then `Last Call` statuses before going to `Final`. See [EIP-1](https://eips.ethereum.org/EIPS/eip-1#eip-process).

Reply:
Should I update the PR to change the status to `Review` then?

Reply:
Yes.